src/vuestic-theme/vuestic-components/va-rating/VaStar.vue

```diff
@@ -0,0 +1,133 @@
+<template>
```

Comment:
Not seeing a demo for this component.

Also naming is incorrect. Should be `va-rating-item` or something similar.